### PR TITLE
Group Bys, but no Schema Inference yet.

### PR DESCRIPTION
**Pull request by vqtran**
Group by's theoretically works in these tests, still need to see if Alexa is happy with our blow-up in sample utterances.

Schema Inference doesn't involve group by column yet, I'll leave it up to the PR that does the Schema Inference fix.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -85,7 +85,7 @@
     <dependency>
       <groupId>com.facebook.presto</groupId>
       <artifactId>presto-parser</artifactId>
-      <version>0.126</version>
+      <version>0.128</version>
```
Comment (vqtran) [inline pom.xml]: Turns out presto-parser is actually far far far far away from being a stable library!

**Comment by GabeLoins** [comment]
wow this some seriously awesome progress

**Comment by GabeLoins** [comment]
ok to merge it

**Comment by GabeLoins** [comment]
im going to add ssb data set noow

**Comment by vqtran** [comment]
Merging, but if you're looking to update your utterances be sure to comment out 90% of the group-by examples until we get a fix

**Comment by GabeLoins** [comment]
err could you expand on that?

**Comment by GabeLoins** [comment]
which 90%?

**Comment by vqtran** [comment]
lets say for now group-bys don't actually work in practice, basically comment out all of those examples. if you want to play around with it you can keep the group-by utterances that don't have any where clauses, and it'll work for like "how many sales are there by store"

**Pull request merged**